src/deps/tinycthread.c

```diff
@@ -20,8 +20,6 @@ freely, subject to the following restrictions:
     3. This notice may not be removed or altered from any source
     distribution.
 */
-
-#if defined(STDC_NO_THREADS)
```

Comment:
the code in the c file has to be excluded or it will be compiled in both case.

Reply:
Removed defines check in both tinycthreads files.